queries.md

```diff
@@ -39,11 +39,13 @@ Limit: { 10 }
 
 <!-- Your Code Goes Here -->
 Filter: {partners: {$exists: true, $size: 0}}
+<!--or-->
+Filter: {partners: {$nin:[]}}
```

Comment:
**Query**: `{$and: [{'ipo.valuation_amount': {$gte: 10000000}}, {founded_year:{$lt: 2010}}]}`

  - Shorthand: `{founded_year: {$lt:2010}, "ipo.valuation_amount": {$gt: 100000000}}`

- **Projection**: `{name: 1, _id: 0, ipo: 1}`